.github/README.md

```diff
@@ -12,18 +12,18 @@
 
 # Downloads
 
-Looking for the most stable or most latest avaliable download? look at the table below!
+Looking for the most stable or the latest avaliable download? Both the latest stable release and the most cutting edge version are in the table below!
```

Comment:
Also just noticed it:
`avaliable` -> `available`
Personally think this should get a little rewritten due to the first sentence essentially saying the same thing as the second. Maybe more like:
`Looking for the latest stable or the most cutting edge download? Look in the table below!` ? Or just ditch the first sentence entirely?
Maybe just `Both the latest stable release and the most cutting edge version can be downloaded from the table below!` ?